tests/phpunit/tests/editor/classic-to-block-menu-converter.php

```diff
@@ -214,7 +214,7 @@ public function test_returns_empty_array_for_menus_with_no_items() {
 
 		$this->assertEmpty( $blocks, 'Result should be empty.' );
 
-		$this->assertIsArray( $blocks, 'Result should be empty array.' );
+		$this->assertIsString( $blocks, 'Result should be empty string.' );
```

Comment:
```suggestion
		$this->assertSame( '', $blocks, 'Result should be empty string.' );
```

Reply:
Done in [0f530d1](https://github.com/WordPress/wordpress-develop/pull/4858/commits/0f530d1e039a15973cce67af4d2163002524303d).